src/com/workspace/utill/LogWriter.java

```diff
@@ -1,22 +1,22 @@
 package com.workspace.utill;
-
+import com.workspace.io.*;
 public class LogWriter {
+    private ConsoleWriter consoleWriter = null;
+    private FileWriter fileWriter = null;
+    public LogWriter(){
+        consoleWriter = new ConsoleWriter();
+        fileWriter = new FileWriter();
+    }
     public void write(Level level, String message){
         switch (level){
             case INFO:
-                writeToConsole("Writing to console: " + message);
+                consoleWriter.write(message);
```

Comment:
Why are we logging the info messages in console and error in file? It should log both error and infos to whatever options use select to log to.

Reply:
right. this will be taken up in next iteration